pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+
+### Checklist
+- [ ] All `TODO`'s are done and removed
+- [ ] No linter errors
+- [ ] Updated docs
+- [ ] `npm run docs` runs successfully
+- [ ] Docker image builds (run `npm run compile` then `docker-compose up`)
+
+#### Changes of Interest
+
```

Comment:
Maybe write an italics description of what we are looking in these sections below